plugins/defaultplugins/aclplugin/vppcalls/acl_vppcalls_test.go

```diff
@@ -0,0 +1,501 @@
+package vppcalls
+
+import (
+	"testing"
+	"github.com/ligato/vpp-agent/tests/vppcallmock"
+	acl_api "github.com/ligato/vpp-agent/plugins/defaultplugins/common/bin_api/acl"
+	"github.com/ligato/vpp-agent/plugins/defaultplugins/common/model/acl"
+	"github.com/ligato/cn-infra/logging/logrus"
+	. "github.com/onsi/gomega"
+)
+
+var acl_NOrules = []*acl.AccessLists_Acl_Rule {}
```

Comment:
you do not have to declare slice via literal. It means you can write 'var anyVar []*acl.AccessLists_Acl_Rule'
also please rather use camel case and apply go fmt to every modified file